### PR TITLE
Add link anchor to team and subteam headings

### LINKED ISSUE #900: anchors for the teams and working groups on governance page
# Technical Feature Request
## Summary
Add some kind of HTML anchor to permit linking directly to a working group from governance pages.

## Motivation
I would like to link directly to the wg-triage team found on [the operations team page](https://www.rust-lang.org/governance/teams/operations), but I cannot! I would like to be able to do `https://www.rust-lang.org/governance/teams/operations#wg-triage`.

## Drawbacks
I can't think of a reason. Perhaps a concern about link rot as team structure changes? 

## Rationale and alternatives
N/A

## Maintenance
No real ongoing maintenance required.

## Unresolved Questions
None

### LINKED ISSUE #1638: Add anchors to sub-teams on team pages
### Summary

It would be nice to be able to link to sub-teams on each team page so one could share a link to e.g. **Library contributors** under [**Library team**](https://www.rust-lang.org/governance/teams/library).

### Motivation

This will make it easier for us to celebrate the work of these subteams and their members!

### Drawbacks

None that I see.

### Rationale and alternatives

Could be a case of "ain't broke don't fix" or "less is more," but I don't think so. :)

### Maintenance

Shouldn't be any.

### Unresolved Questions

None that I have.

### PR DESCRIPTION
**Pull request by pierwill**
This is to close #1638 (and maybe #900, too).

### CODE REVIEW

**Comment by Manishearth** [comment]
I can approve this once you fix the hover style; probably just give it a class and add an `a.className:hover` rule

File: src/styles/app.scss [hunk 1]
```diff
@@ -382,6 +382,10 @@ footer {
   }
 }
 
+a.sub {
```
Comment (Manishearth) [inline src/styles/app.scss:385]: nit: please use a more descriptive class name, probably linkable-subheading

Comment (pierwill) [reply src/styles/app.scss]: Fixed 👍

**Review by Manishearth — APPROVED** [review]

**Comment by Manishearth** [comment]
Thanks!

**Pull request merged**